nservicebus/wormhole/index.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Wormhole Gateway
+summary: How to send messages between geographically separate sites using Worm Hole Gateway 
+component: Wormhole
+reviewed: 2017-05-20
+---
+
+`NServiceBus.Wormhole` is an alternative to the [Gateway](/nservicebus/gateway/) component of NServiceBus. Following table compares the most important aspects of both technologies.
+
+|                         | Gateway                                 | Wormhole                                                                                |
+|-------------------------|-----------------------------------------|-----------------------------------------------------------------------------------------|
+| Encryption              | transport-level                         | no                                                                                      |
+| Transports              | <ul><li>HTTP</li><li>custom</li></ul>   | <ul><li>all NServiceBus transports</li><li>HTTP</li></ul>                               |
+| Topology                | gateway-in-endpoint                     | gateway-per-site                                                                        |
+| Routing                 | set destination site when sending       | <ul><li>set destination site via routing config</li><li>set destination site from message content</li></ul> |
+| Failure handling        | <ul><li>immediate retries</li><li>delayed retries</li></ul> | <ul><li>immediate retries</li></ul>                                 |
+| Automatic deduplication | built-in via persistence                | no                                                                                      |
+|                         |                                         |                                                                                         |
+|                         |                                         |                                                                                         |
+|                         |                                         |                                                                                         |
+
+## Topology
+
+The Wormhole-enabled system consists of *sites*. Sites can be geographically distributed. It is assumed that the connectivity within the site is much better than between the sites. Each site forms an independent NServiceBus system with its own set of infrastructure e.g. ServiceControl and ServicePulse.
```

Comment:
set of infrastructure, for example, local instances of ServiceControl and ServicePulse.

Link the products to the doco pages.